.github/workflows/build.yml

```diff
@@ -33,10 +33,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Install Node.js 14.x
+      - name: Install Node.js 14.18.1
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: '14.18.1'
```

Comment:
Is it now mandatory to use this specific version of Node.js for development?

If so, it should be documented here:

https://github.com/arduino/arduino-ide/blob/130586e0f40b46d4beef4065fbc9e713ae5d712d/BUILDING.md#L45

Reply:
Can you please help, @msujew? Thank you!

Reply:
I don't think it's necessary. I had trouble getting the CI to complete, and setting the node version to a specific version was one of the approaches I had taken to ensure I had a reproducible environment. @kittaakos You can safely revert the change.

Reply:
I appreciate your help.